src/Commands/GenerateSitemap.php

```diff
@@ -0,0 +1,148 @@
+<?php
```

Comment:
Will this command always finish? Or run into a timeout maybe? We have a project where there's 23K entries (divided over multiple shops) of a collection that should be in the sitemap. Maybe create Jobs for each sitemap so we can schedule this in a queue and do it in smaller operations? Also would e nice to be able to schedule this once or twice per day.